cmd/pullrequest-init/disk_test.go

```diff
@@ -113,26 +115,39 @@ func TestToDisk(t *testing.T) {
 			t.Errorf("Get Status: -want +got: %s", diff)
 		}
 	}
-
 	// Check the labels
 	fis, err = ioutil.ReadDir(filepath.Join(d, "labels"))
 	if err != nil {
 		t.Fatal(err)
 	}
 	labels := map[string]struct{}{}
+	labelManifest := Manifest{}
 	for _, fi := range fis {
+		if fi.Name() == manifestPath {
+			continue
+		}
 		text, err := url.QueryUnescape(fi.Name())
 		if err != nil {
 			t.Errorf("Error decoding label text: %s", fi.Name())
 		}
 		labels[text] = struct{}{}
+		labelManifest[text] = true
 	}
 
 	for _, l := range tektonPr.Labels {
 		if _, ok := labels[l.Text]; !ok {
 			t.Errorf("Expected label with text: %s, not found: %v", l.Text, labels)
 		}
 	}
+	gotManifest := readManifest(t, filepath.Join(d, "labels", manifestPath))
+	for _, m := range gotManifest {
+		if _, ok := labelManifest[m]; !ok {
+			t.Errorf("Label %s not found in manifest", m)
+		}
+	}
+	if len(labelManifest) != len(gotManifest) {
+		t.Errorf("Label manifest does not match expected length. %d !=  %d, got %+v", len(labelManifest), len(gotManifest), gotManifest)
```

Comment:
suggest "expected %d, received %d", just to make it clear which number was the assertion and which was the result.

Reply:
Done.